src/Helper/ValidateStreet.php

```diff
@@ -60,14 +60,15 @@ public static function validate(string $fullStreet, string $localCountry, ?strin
      */
     public static function getStreetRegexByCountry(string $local, string $destination): ?string
     {
-        if (
-            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_NL) ||
-            ($local === AbstractConsignment::CC_NL && $destination === AbstractConsignment::CC_NL)
-        ) {
+        $localIsNlOrBe = in_array($local, [AbstractConsignment::CC_BE, AbstractConsignment::CC_NL]);
+        $destinationIsNl = AbstractConsignment::CC_NL;
+        $destinationIsBe = AbstractConsignment::CC_BE;
+
+        if ($localIsNlOrBe && $destinationIsNl) {
```

Comment:
- [ ] Dit alles graag even laten testen door een ander; in Wordpress of Magento
(Het is nogal een belangrijk onderdeel)